### DISCUSSION TITLE
cpu rises and falls

### DISCUSSION THREAD

**Discussion by MurphyDoe0815** [dani-garcia/vaultwarden#3118] [Q&A]
Hello all,

I am using the latest server with a turnkey core on proxmox.
unfortunately i have problems with the cpu that keeps going up and down. on other docker containers that are running next door there is nothing !
i have installed htop and there i could always get the message 

/lib/systemd/systemd-resolved 
/lib/systemd/systemd-networkd
/lib/systemd/systemd-journald

here you can see the up and down of the cpu every second.

any idea ?

**Comment by BlackDex** [comment]
If your cou problems are with the `systemd` applications, I'm unsure why you are reporting this here. Vaultwarden doesn't use anything it self from systemd. Only link could be syslog if you have that enabled in the config. But other then that, Vaultwarden stands on its own.

Some logs of Vaultwarden or other logs you think might be relevant would be useful to post maybe. But not sure if we can help you with this specific problem.
